docs/USB Flashing.md

```diff
@@ -49,13 +49,21 @@ With the board connected and in bootloader mode (reset it by sending the charact
 * Choose Options > List All Devices
 * Select `STM32 BOOTLOADER` in the device list
 * Choose `WinUSB (v6.x.x.x)` in the right hand box
-
+   
 ![Zadig Driver Procedure](assets/images/zadig-dfu.png)
 
 * Click Replace Driver
 * Restart the Configurator (make sure it is completely closed, logout and login if unsure)
 * Now the DFU device should be seen by Configurator
 
+## Platoform: Mac-OS
```

Comment:
That is a really good point that I did not think of 😃. Should I just delete the branch or rephrase it from Mac-OS  to something along the lines of “while using USB-C”

Reply:
Just edit it. :)

Reply:
@mmosca by any chance do you connect your USB-C to C to a USB a dongle or hub? I did some research and found out USB-C only sends power when the device requests it while USB-A essentially always sends power

Reply:
USB-C can be used on both the host and device side. It can also support other (higher) voltages and power levels, so it needs a bit more complicated stuff on both sides of the connection to successfully negotiate the type of connection (host/device) and power levels.